README.md

```diff
@@ -31,7 +39,8 @@ Which *one* of the following buttons would *not* be displayed?
 [ ] A: <button data-ng-if="status">Click</button>
 [ ] B: <button data-ng-show="status">Click</button>
 [ ] C: <button data-ng-hide="status">Click</button>
```

Comment:
since the value "click" is not falsy, `ng-hide` will evaluate to true and thus not display the button